docs/proposals/28-autonomous-shoot-clusters.md

```diff
@@ -0,0 +1,495 @@
+   If there is no existing Gardener system yet, the needed configuration must be provided manually.
+2. The downloaded configuration and a `Shoot` manifest are provided to the `gardenadm init` command.
+   This will bootstrap the first control plane node.
+3. Optionally, additional control plane nodes can be joined.
+   In order to do so, the user runs `gardenadm token generate` which generates a random bootstrap token, and then `gardenadm token create <token>` which creates it on the server.
+   Now connected to the new node that shall be joined, the user runs the `gardenadm join` command which bootstraps another control plane node.
+5. Optionally, worker nodes can be joined.
+   The flow is the same - a bootstrap token must be generated and created, and then `gardenadm join` must be executed to bootstrap the worker node.
+6. Once above steps are completed, the user runs `gardenadm connect` to deploy a `gardenlet` into the cluster which connects to an existing Gardener system for further cluster management.
+
+A more detailed description of the commands is following.
+
+#### `gardenadm discover`
+
+This command takes a kubeconfig to an existing Gardener system, and a `Shoot` manifest.
+It downloads the required `CloudProfile`, `ControllerRegistration` and `ControllerDeployment` resources and stores them locally for usage of other `gardenadm` commands.
+This simplifies the crafting of these documents, and it also ensures that the configuration and versions used for the bootstrapping of the autonomous shoot cluster fit to the configuration later found in the connected Gardener system.
+
+#### `gardenadm init`
+
+This command bootstraps the first control plane node of the autonomous shoot cluster.
+The configuration is specified via separate files.
+It needs a `Shoot` manifest and some additional configuration files, which usually reside in the garden cluster, e.g., a `CloudProfile` (best downloaded with `gardenadm discover`).
+For the initial cluster Gardener bootstrapping use-case however, these won't come from a pre-existing garden cluster as it doesn't exist yet.
+They can then be provided locally (that's why `gardenadm discover` and `gardenadm init` are separate commands), reflecting the to-be-deployed Gardener system later.
+
+The high-level steps executed by the command are as follows:
+
+1. Generate certificate authorities, certificates and keys using the [secrets manager](../development/secrets_management.md).
+2. Create [`OperatingSystemConfig` resource](../extensions/operatingsystemconfig.md) containing `systemd` units and files.
+   This includes the files for the minimal standard Kubernetes control plane, i.e., `kubelet`, `etcd`, `kube-apiserver`, `kube-controller-manager`, and `kube-scheduler`.
+4. Apply the `OperatingSystemConfig` resource to the node using the [`gardener-node-agent`](../concepts/node-agent.md).
```

Comment:
```suggestion
3. Apply the `OperatingSystemConfig` resource to the node using the [`gardener-node-agent`](../concepts/node-agent.md).
```

Technically should be the third item.

Reply:
Apparently, we changed the entries of two lists and have incorrect numbering there. This will be fixed in the next commit.